lib/matplotlib/axes/_base.py

```diff
@@ -3995,9 +3995,10 @@ def _get_view(self):
             default implementation saves the view limits. You *must* implement
             :meth:`_set_view` if you implement this method.
         """
-        xmin, xmax = self.get_xlim()
-        ymin, ymax = self.get_ylim()
-        return xmin, xmax, ymin, ymax
+        return {
```

Comment:
This changes the return type, which by the note above

> Intended to be overridden by new projection types

could be a compatibility problem. Just to be sure I understand this correctly: The return type only needs to be pairwise compatible with the respective `_set_view` ("You *must* implement´:meth:`_set_view` if you implement this method.). And therefore we're safe.

Maybe we should mention this relation so that authors of new projections know they can define their own data as needed.

Reply:
Adjusted the docstrings, how do they look to you?  In my view the return value of _get_view is really an opaque object, just something that needs to be comprehensible by the corresponding _set_view.